### PR TITLE
fix i18n

### PR DESCRIPTION
**Pull request by viktorhaid**
Adds some fixes to get translations working, which were introduced in https://github.com/pinterest/snappass/pull/330.

Additionally it compiles the .mo files during `docker build`.

### CODE REVIEW

File: snappass/main.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 from urllib.parse import unquote_plus
 from urllib.parse import urljoin
 from distutils.util import strtobool
-from flask_babel import Babel
+from flask_babel import Babel, _
```
Comment (yurushao) [inline snappass/main.py:14]: why importing `_`?

Comment (yurushao) [reply snappass/main.py:14]: this caused linter error

Comment (viktorhaid) [reply snappass/main.py:14]: I didn't manage to get the Jinja templates translated without importing `_`. Came accross the solution [here](https://blog.miguelgrinberg.com/post/the-flask-mega-tutorial-part-xiii-i18n-and-l10n).

Should i [in-line ignore](https://flake8.pycqa.org/en/stable/user/violations.html#in-line-ignoring-errors) this linter error? Other suggestions?

Comment (yurushao) [reply snappass/main.py:14]: Got it. In-line ignoring is fine, please add explanation why doing that.

**Review by yurushao — APPROVED** [review]

**Comment by systeembeheerder** [comment]
@viktorhaid This fixes only the docker version?

as i wrote in #330 
I guess the compilation of the .mo files should be some part of the build process?

**Pull request merged**